values.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 image:
   repository: gcr.io/planet-4-151612/robots
-  tag: v0.1
+  tag: latest
```

Comment:
Using tag:latest for docker images won't necessarily force a redeploy when the image changes, better to pin to actual releases.